### PR TITLE
2.x - Fix tests

### PR DESCRIPTION
**Pull request by markstory**
* Update tests to PHPUnit 8
* Update to latest codesniffer rules.

### CODE REVIEW

File: tests/test_app/TestApp/Http/TestRequestHandler.php [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
  * Licensed under The MIT License
  * For full copyright and license information, please see the LICENSE.txt
  * Redistributions of files must retain the above copyright notice.
  *
- * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- * @link          http://cakephp.org CakePHP(tm) Project
- * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ * @copyright Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link http://cakephp.org CakePHP(tm) Project
+ * @license http://www.opensource.org/licenses/mit-license.php MIT License
```
Comment (markstory) [inline tests/test_app/TestApp/Http/TestRequestHandler.php]: Rector did this, and I'm too lazy to fight it.

**Comment by ADmad** [comment]
I'll take care of the phpstan failures.

File: tests/test_app/TestApp/Http/TestRequestHandler.php (hunk 1 above)
Comment (ADmad) [reply tests/test_app/TestApp/Http/TestRequestHandler.php]: There are more important battles to fight :slightly_smiling_face:.

Comment (markstory) [reply tests/test_app/TestApp/Http/TestRequestHandler.php]: That was my feeling too.

**Pull request merged**